h2o-algos/src/main/java/hex/klime/KLimeModel.java

```diff
@@ -27,6 +27,11 @@ public KLimeModel(Key<KLimeModel> selfKey, KLimeParameters params, KLimeOutput o
   }
 
   @Override
+  public GridSortBy getDefaultGridSortBy() {
```

Comment:
Good to know. I reviewed Leaderboard now, there is some overlap but unfortunately I don't see how we could unify these two approaches.

Grid Search is meant for a single type of model, it is possible that two different kinds of models will have different preferred sort-by (eg. k-LIME is a regression model technically but we don't want to use the default for regression models).

I think this API is okay, not duplicate.